proto/cosmos/group/v1beta1/types.proto

```diff
@@ -174,98 +171,101 @@ message Proposal {
   // When a decision policy is changed, existing proposals from previous policy versions will become invalid.
   uint64 group_policy_version = 7;
 
-  // Status defines proposal statuses.
-  enum Status {
-    option (gogoproto.goproto_enum_prefix) = false;
+  // status represents the high level position in the life cycle of the proposal. Initial value is Submitted.
+  ProposalStatus status = 8;
 
-    // An empty value is invalid and not allowed.
-    STATUS_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "ProposalStatusInvalid"];
+  // result is the final result based on the votes and election rule. Initial value is unfinalized.
+  // The result is persisted so that clients can always rely on this state and not have to replicate the logic.
+  ProposalResult result = 9;
```

Comment:
With `gov` status and result are the same thing right?

Reply:
Yeah. IIUC, we have:
- if status==STATUS_CLOSED, then result==RESULT_ACCEPTED or RESULT_REJECTED
- if status is anything else, then result==RESULT_UNFINALIZED

i'm not sure why the initial design was like this, but IMO I think we could consider removing `STATUS_CLOSED` and `result`, and adding `STATUS_PASSED` and `STATUS_REJECTED` just like gov.